src/main/resources/avro/common.avdl

```diff
@@ -353,6 +353,69 @@ record Sequence {
 }
 
 /**
+An abstraction for referring to a genomic region, in relation to some already 
+known reference. This will require some significant rework as we move to graph
+coordinates.  
+*/
+// TODO: Add support for paths through the graph.
+record Region {
+  /**
+  The starting point of the region. For the moment the base-pair included in
+  the region, on the 5' end of the oriented region.
+  */
+  Position start;
+
+  /**
+  The length, in base pairs
+  */
+  long length;
+}
+
+/**
+An ontology term describing an attribute. (e.g. the phenotype attribute
+'polydactyly' from HPO)
+*/
+record OntologyTerm {
+  /**
+  The source of the onotology term.
+  (e.g. `Ontology for Biomedical Investigation`)
+  */
+  string ontologySource;
+
+  /**
+  The ID defined by the external onotology source.
+  (e.g. `http://purl.obolibrary.org/obo/OBI_0001271`)
+  */
+  string id;
+
+  /** The name of the onotology term. (e.g. `RNA-seq assay`) */
+  union { null, string } name = null;
+}
+
+/**
+Identifier from a public database
+*/
+record ExternalIdentifier {
+  /**
+  The source of the identifier.
+  (e.g. `Ensembl`)
+  */
+  string database;
+
+  /**
+  The ID defined by the external database.
+  (e.g. `ENST00000000000`)
+  */
+  string identifier;
```

Comment:
Good point, but I feel that this is something that needs to be done on master directly. It is a bit out of the scope of the pull request.